index.js

```diff
@@ -361,6 +352,11 @@ function unified() {
 
     function executor(resolve, reject) {
       var file = vfile(doc)
+      // Process pipeline.
+      var pipeline = trough()
+        .use(pipelineParse)
+        .use(pipelineRun)
+        .use(pipelineStringify)
```

Comment:
Wouldn’t this make every `run` significantly slower?

Reply:
Maybe, benchmarking would give better insights on the potential impact.